docs/notebooks/t-test.ipynb

```diff
@@ -62,18 +62,99 @@
     },
```

Comment:
<p>Mention the <code>0</code>  removes the intercept. Using <code>-1</code>  is also equivalent.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bambinos/bambi/pull/351/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>